Cakes-Co/index.html

```diff
@@ -1,16 +1,143 @@
+                Welcome to Rb's Cakes, the ultimate destination for cupcake
+                connoisseurs. Our delectable cupcakes are a sensory delight,
+                offering a wide range of flavors to satisfy your every craving.
+                From timeless classics like rich chocolate and creamy vanilla to
+                unique, imaginative combinations that will enchant your taste
+                buds, we have a treat for every occasion. Whether you're
+                celebrating a special event or simply yearning for a delightful
+                indulgence, Rb's Cakes has you covered. Explore our selection,
+                place your order, and experience the joy of Rb's Cakes, where
+                happiness is just a bite away!
+              </p>
+            </div>
+          </article>
+        </section>
+        <section class="cupcake-group">
+          <div class="cupcake-container">
+            <img src="img/cupcake-1.webp" alt="" class="cupcake_item-img" />
+            <div class="middle-btn">
+              <a class="menu_item" href="#">Fruity</a>
+            </div>
+          </div>
+
+          <div class="cupcake-container">
+            <img src="img/cupcake-2.webp" alt="" class="cupcake_item-img" />
+            <div class="middle-btn">
+              <a class="menu_item" href="#">Chocolatey</a>
+            </div>
+          </div>
+
+          <div class="cupcake-container">
+            <img src="img/cupcake-3.webp" alt="" class="cupcake_item-img" />
+            <div class="middle-btn">
+              <a class="menu_item" href="#">Hallowey</a>
+            </div>
+          </div>
+          <div class="cupcake-container">
+            <img src="img/cupcake-4.webp" alt="" class="cupcake_item-img" />
+            <div class="middle-btn">
+              <a class="menu_item" href="#">Coloury</a>
+            </div>
+          </div>
+        </section>
+      </main>
+
+      <!-- Footer-->
+      <footer class="footer">
+        <!-- Social Media Logos -->
+        <div class="social-media">
+          <div class="social-media-item">
+            <div class="circle">
+              <img src="img/logo-x-twitter.svg" alt="x-icon" />
+            </div>
+
+            <p>@rb_cakes_mcr</p>
+          </div>
+          <div class="social-media-item">
+            <div class="circle">
+              <img src="img/facebook-icon.svg" alt="facebook-icon" />
+            </div>
+            <p>@rb_cakes_mcr</p>
+          </div>
+          <div class="social-media-item">
+            <div class="circle">
+              <img src="img/instagram-icon.svg" alt="instagram-icon" />
+            </div>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+            <p>@rb_cakes_mcr</p>
+          </div>
+        </div>
+        <!-- Copyright -->
+        <div class="copyright">&copy; Rb's Cake, Ltd.</div>
+      </footer>
+    </div>
 
-</html>
\ No newline at end of file
+    <script>
+      function switchMenu() {
```

Comment:
I can't really complain too much beacause it seems to be working but at this point in the course we normally stick to a CSS and HTML only solution, so while I think it's good that you've gone and found this, I'd really like to see this redone without the JS